### ISSUE TITLE
tsgo diagnostics are dropped for SvelteKit route group paths

### ISSUE THREAD

**Issue by pheuter** [pheuter/svelte-check-rs#124]
## Summary

When tsgo reports a diagnostic for a file under a SvelteKit route group, the diagnostic path contains parentheses, for example:

```text
src/routes/(app)/+page.svelte.ts(10,5): error TS2322: ...
```

The tsgo output parser looked for the first `(` in the line and treated it as the diagnostic position. For route-group paths, that means it tries to parse the route-group directory as the line/column segment and drops the diagnostic entirely.

## Expected behavior

Diagnostics for files under route groups should be parsed and mapped back to the original `.svelte` file like diagnostics for any other path.

## Coverage needed

Add parser coverage for tsgo output lines with parentheses in the path, plus an integration fixture that verifies a TypeScript error inside a route-group page is reported.
